src/Umbraco.Core/PropertyEditors/ValueConverters/JsonValueConverter.cs

```diff
@@ -28,13 +31,16 @@ public JsonValueConverter(PropertyEditorCollection propertyEditors)
 
         /// <summary>
         /// It is a converter for any value type that is "JSON"
+        /// Unless its in the Excluded Property Editors list
+        /// The new MediaPicker 3 stores JSON but we want to use its own ValueConvertor
         /// </summary>
         /// <param name="propertyType"></param>
         /// <returns></returns>
         public override bool IsConverter(IPublishedPropertyType propertyType)
         {
             return _propertyEditors.TryGet(propertyType.EditorAlias, out var editor)
-                   && editor.GetValueEditor().ValueType.InvariantEquals(ValueTypes.Json);
+                   && editor.GetValueEditor().ValueType.InvariantEquals(ValueTypes.Json)
+                   && Array.IndexOf(ExcludedPropertyEditors, propertyType.EditorAlias) == -1;
```

Comment:
Is there a reason why we don't use something like: `ExcludedPropertyEditors.Contains(propertyType.Alias);` instead of: `Array.IndexOf(ExcludedPropertyEditors, propertyType.EditorAlias) == -1;`, it should be functionally the same, but would, in my opinion, be a bit more readable.